integration/docker/run_test.go

```
@@ -23,6 +24,21 @@ func withWorkload(workload string, expectedExitCode int) TableEntry {
 	return Entry(fmt.Sprintf("with '%v' as workload", workload), workload, expectedExitCode)
 }
 
+func distroID() string {
+	pathFile := "/etc/os-release"
+	_, err := os.Stat(pathFile)
+	if _, err := os.Stat(pathFile); os.IsNotExist(err) {
+		pathFile = "/usr/lib/os-release"
+	}
+	cmd := exec.Command("sh", "-c", fmt.Sprintf("source %s; echo -n $ID", pathFile))
+	id, _:= cmd.CombinedOutput()
```

you're ignoring the error